rackcity/models/log.py

```diff
@@ -0,0 +1,30 @@
+from .it_model import ITModel
+from django.contrib.auth.models import User
+from django.db import models
+from .asset import Asset
+
+
+class Log(models.Model):
+    date = models.DateTimeField(
+        auto_now=True,
+        blank=True,
+        editable=False,
+    )
+    log_content = models.TextField()
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        verbose_name="related user",
+    )
+    related_assets = models.ManyToManyField(
+        Asset,
+        verbose_name="related assets",
+        blank=True,
+    )
```

Comment:
Will on_delete default to something here? Or is that not a thing on many to many fields

Reply:
many to many does not have on_delete. upon investigation i have come to learn that many to many delete behavior is similar to cascade. once i actually start creating these logs i might come back and edit this field.......tbd

Reply:
If you're concerned about this you could have a `related_asset` and `other_related_asset`  field each as a foreign key... haha jk... unless?

Reply:
I tried this too LOL and you can't have multiple foreign keys to the same model that look the same


haha unless.